### ISSUE TITLE
Windows Installation Issues

### ISSUE THREAD

**Issue by straussmaximilian** [MannLabs/CKG#30]
**Describe the bug**
Hi,
I tried to follow the Windows installation guide and tried an install on a fresh Windows machine.
OS: Microsoft Windows Server 2019

The first errors that occurred were several `SSL: CERTIFICATE_VERIFY_FAILED` when installing the requirements via pip. 
I stumbled across this [thread](https://github.com/pypa/pip/issues/4205) and could imagine that this might be related to Python 3.6.  I was able to get all packages installed by manually installing the packages that raised the error (i.e. `pip3 install --ignore-installed -r requirements.txt` > Error > `pip3 install error_package` > `pip3 install --ignore-installed -r requirements.txt` > Error > `pip3 install error_package` > `pip3 install --ignore-installed -r requirements.txt`.

After the requirements install went through, I went to the celery installation. This then fails as no matching distribution can be found. Is this meant to be installed with `pip` or `pip3`? As it is stated to be `pip` and all the other installs are with `pip3`.

Installation of a newer celery version `celery-4.4.7` works, but I am wondering if this is causes problems at a later stage as indicated in the documentation.
Attached is the environment that is not compatible with the celery install.

I can imagine that solving environment/package issues might take while (even maybe switching to a newer Python version), so maybe it is worth investigating the celery functionality issue. Any suggestions on that? Then I might be able to investigate.

[env_export.txt](https://github.com/MannLabs/CKG/files/5012669/env_export.txt)
<img width="899" alt="Screen Shot 2020-08-02 at 20 52 59" src="https://user-images.githubusercontent.com/16521288/89130482-861fb880-d505-11ea-9c31-8f21f1ff8fc6.png">

**Comment by arcolaco** [comment]
Hi Max,

Thanks for your report.
I have tested the installation on a Windows 10 machine and haven't seen the  SSL: CERTIFICATE_VERIFY_FAILED so it might be specific to your system. We will make a note in the documentation if other users run into the same issue.

Regarding the celery installation, the version pulled from the pip repository doesn't seem to be compatible with Windows. That's why we clone the latest version directly form the Git repository. I have tested both pip and pip3 and there are no issues arising later on when trying to create a project, for example.
We will modify the docs and change it to pip3 for coherence.
